scripts/jenkins/ardana/ansible/vars/main.yml

```diff
@@ -8,3 +8,14 @@ rpm_verification_fixmes: "{{ verification_temp_dir }}/rpm-verify-modification-wh
 rpm_modifiable_config_files: "{{ verification_temp_dir }}/rpm-modifiable-config-files.txt"
 rpm_modifiable_config_file_regexps: "{{ verification_temp_dir }}/rpm-modifiable-config-file-regexps.txt"
 rpm_added_files_whitelist: "{{ verification_temp_dir }}/rpm-added-files-whitelist.txt"
+
+ca_file: "{{ trusted_ca_bundle }}"
+admin_project_name: "{{ KEY_API.vars.keystone_service.tenant }}"
+
+glance_admin_user: "{{ GLA_API.consumes_KEY_API.vars.keystone_glance_user }}"
+glance_auth_url: "{{ GLA_API.consumes_KEY_API.vips.private[0].url }}"
+glance_admin_password: "{{ GLA_API.consumes_KEY_API.vars.keystone_glance_password | quote }}"
```

Comment:
Does this need to be defined in vars? I am generally reluctant to have password in vars/
On top of it, it could be used directly in the play, as this is only used once, and I don't expect any override. But I might be wrong.

Reply:
I can try that but this is also how it is done in most other playbooks that I checked, like neutron-ansible

Reply:
These were general recommendations, no obligation to follow them : )